app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+    validates :username, uniqueness: true
```

Comment:
- [ ] username is validated (at least for the presence)

Reply:
Perfect I changed it for two

Reply:
perfect